src/main/java/org/folio/circulation/services/FeeFineFacade.java

```diff
@@ -88,7 +88,8 @@ public FeeFineFacade(Clients clients) {
   }
 
   private CompletableFuture<Result<Void>> refundAndCloseAccount(RefundAccountCommand refund) {
-    final AccountRefundContext context = new AccountRefundContext(refund.getAccountToRefund());
+    final var context = new AccountRefundContext(refund.getAccountToRefund(),
```

Comment:
Could `val` be used rather than `final var`?

```suggestion
    val context = new AccountRefundContext(refund.getAccountToRefund(),
```

Reply:
sure, done